jxbrowser/pom.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
@vladimir-ikryanov, what indentation level do they standardly use in their projects?

The current one looks too odd and big to me.